docs/source/conf.py

```diff
@@ -74,13 +74,13 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "alabaster"
+html_theme = "sphinx_rtd_theme"
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
-# html_theme_options = {}
+html_theme_options = {"display_version": False}
```

Comment:
@davelab6 I intentionally set the version display to `False` because RTD adds the git commit short sha1 hash string at the bottom of the documentation (and because it is a PIA to remember to bump the docs version each time you release a software version).  I don't know if RTD renders version numbers on the remote builds if this is set back to `True` (default, can just comment this out), but you can try this if you'd like to print version number on the documentation.  The PyPI release version is present in the badge on the home page of the docs.

Associated PR: #2154

Reply:
> it is a PIA to remember to bump the docs version each time you release a software version

Then the automated release process should make it happen automatically :)

We have fairly frequent issues posted for things that were already fixed on master but yet to be released to pypi or the user didnt update, and I think if the RTD docs are always matching what's on master, not whats on pypi, and we expect most users to be using the pypi version, then it may be better to display the version, and now the sha1... 

Not too strong on this tho

Reply:
Is the version being pulled from git tags?  I haven't used the automated `use_scm_version` for packaging.